### PR TITLE
Issues fixed

### PR DESCRIPTION
**Pull request by meghanstovall**
Fixed some issues with our game class and how we were reading the data. This helped with passing more of the methods in the spec harness. Also fixed the deleted methods that got lost in the merge conflict. Fixed biggest bust method to also pass the spec harness.

### CODE REVIEW

**Review by holmesm8 — APPROVED** [review]
I like the updated methods as well as the cleaning up of the extra some we don't need (like a mini refactor). Great job removing all of the test variables we had set so that it looks much cleaner.

**Comment by KellyIB** [comment]
Hey, nice job on getting the bust method to pass the spec harness.  Coverage has changed, we should take a look after the next pull request's merge conflicts have been resolved.

**Pull request merged**
